src/Microsoft.AspNet.Routing/RouteDirection.cs

```diff
@@ -3,9 +3,19 @@
 
 namespace Microsoft.AspNet.Routing
 {
+    /// <summary>
+    /// Indicates whether ASP.NET routing is processing a URL from a client or generating a URL.
+    /// </summary>
     public enum RouteDirection
     {
+        /// <summary>
+        /// A URL from a client is being processed.
+        /// </summary>
```

Comment:
> A URL from a client

Can we say something about this being the URL of an HTTP request? From a client is a little ambiguous